src/gh.js

```diff
@@ -32,22 +32,23 @@ async function getRegistrationToken() {
 }
 
 async function removeRunner() {
-  const runner = await getRunner(config.input.label);
-  const octokit = github.getOctokit(config.input.githubToken);
+  for (let i = 1; i <= parseInt(config.input.numberOfRunners); i++) {
+    const runner = await getRunner(config.input.label);
+    const octokit = github.getOctokit(config.input.githubToken);
 
-  // skip the runner removal process if the runner is not found
-  if (!runner) {
-    core.info(`GitHub self-hosted runner with label ${config.input.label} is not found, so the removal is skipped`);
-    return;
-  }
+    // skip the runner removal process if the runner is not found
```

Comment:
Indentation is not correct at here I guess.

Reply:
It is correct, because it is under the for loop defined in line 35.